spec/github_fetcher_spec.rb

```diff
@@ -60,8 +60,8 @@
 
   let(:team_config) do
     {
-      'language' => nil,
-      'channel' => nil
+      'language' => 'ruby',
+      'channel' => '#ruby'
```

Comment:
Should you also test for the case of an improperly configured YAML file, e.g., one with no languages set?  What happens in that situation, anyway?  Should the bot raise a warning that you have nothing set and therefore it won't actually do anything?

This doesn't have to be considered for this PR, just a thought I had triggered by this change.